odoo/orm/models.py

```diff
@@ -586,7 +592,7 @@ def _add_field(self, name, field):
         # Assert the name is an existing field in the model, or any model in the _inherits
         # or a custom field (starting by `x_`)
         is_class_field = any(
-            isinstance(getattr(model, name, None), fields.Field)
+            isinstance(getattr(model, name, None), Field)
```

Comment:
This is noise, it has nothing to do with domains.  Move that into a separate PR.

Reply:
#189429